src/app/features/psbt-signer/components/psbt-request-details/components/psbt-inputs-and-outputs/components/psbt-output-list/psbt-output-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { PsbtOutput } from '@app/features/psbt-signer/hooks/use-parsed-outputs';
+
+import { PsbtOutputItem } from './components/psbt-output-item';
+import { PsbtOutputListLayout } from './components/psbt-output-list.layout';
+
+interface PsbtOutputListProps {
+  addressNativeSegwit: string;
+  addressTaproot: string;
+  outputs: PsbtOutput[];
+}
+export function PsbtOutputList({
+  addressNativeSegwit,
+  addressTaproot,
+  outputs,
+}: PsbtOutputListProps) {
+  return (
+    <PsbtOutputListLayout>
+      {outputs.map((output, i) => {
+        return (
+          <PsbtOutputItem
+            addressNativeSegwit={addressNativeSegwit}
+            addressTaproot={addressTaproot}
+            key={i}
+            output={output}
+          />
+        );
+      })}
```

Comment:
```suggestion
      {outputs.map((output, i) => (
        <PsbtOutputItem
          addressNativeSegwit={addressNativeSegwit}
          addressTaproot={addressTaproot}
          key={i}
          output={output}
        />
      ))}
```

![](https://gifdb.com/images/high/spin-running-in-circles-homer-simpsons-ch2n7ccgjo44thht.gif)

Reply:
Haha.

Reply:
What does this gif mean here?